content/lang/profile/authCallback.en.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/*
+------------------
+Language: English
+------------------
+*/
+
+$LANG['CAUGHT_EXCEPTION'] = 'Caught exception: ';
+$LANG['LOCATION'] = 'Location:';
```

Comment:
I think that the location tag can be removed entirely from this file (and from the spanish and french versions as well).